layer/nrlf/core/tests/test_validators.py

```diff
@@ -599,22 +599,16 @@ def test_validate_category_coding_invalid_system():
     }
 
 
-def test_validate_content_extension_too_many_extensions():
+def test_validate_author_too_many_authors():
```

Comment:
Did you mean to remove the `test_validate_content_extension_too_many_extensions()` unit test?

Reply:
nope